src/insns/atomic_exceptions.adoc

```diff
@@ -1,11 +1,13 @@
 Permissions::
 ifdef::cap_atomic[]
-Requires the authorising capability to be tagged and not sealed.
+Requires the authorizing capability to be tagged and not sealed.
 +
 Requires <<r_perm>> and <<w_perm>> in the authorising capability.
 +
 If <<c_perm>> is not granted then store the memory tag as zero, and load `cd.tag` as zero.
 +
+If the authorizing capability does not grant <<lm_perm>>, and the tag of `cd` is 1 and `cd` is not sealed, then an implicit <<ACPERM>> clearing <<w_perm>> and <<lm_perm>> is performed to obtain the final permissions on `cd` (see <<LC>>)..
++
 (_This tag clearing behaviour may become a data dependent exception in future._)
```

Comment:
The _this_ in this sentence is a reference to:
> If <<c_perm>> is not granted then store the memory tag as zero, and load `cd.tag` as zero.

So the sentence should really move up, or even better, remove it altogether! It sounds like its still unclear what the behaviour must be here.

Reply:
I think the highlighted sentence reads ok.....